tests/cover/test_simple_characters.py

```diff
@@ -117,3 +122,14 @@ def test_blacklisted_characters():
 
     with pytest.raises(NoSuchExample):
         find(st, lambda c: c in bad_chars)
+
+
+def test_whitelist_characters_disjoint_blacklist_characters():
+    good_chars = u'👍✔'
+    bad_chars = u'👎✘'
+    st = characters(min_codepoint=ord('0'), max_codepoint=ord('9'),
```

Comment:
Nitpicking and you don't have to fix this, but it's slightly weird that this test would pass even if `blacklist_characters` support wasn't working. I think it would probably pass most of the time by accident even if you didn't pass `blacklist_characters`! It might be better to restrict the testing to some small range of codepoints that actually includes some of the blacklisted characters - e.g. you could add a digit to the `blacklist_characters`.